pages/docs/guides/message-validation.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Messages validation
+description: This guide explains different use cases for message validation using AsyncAPI.
+weight: 120
+---
+
+## Introduction
+This guide explains different concepts of validating messages. You will also learn what role AsyncAPI documents play in validation.
+
+## Message validation
+To understand message validation, we must first understand the basic components involved.
+- Producer: The producer is responsible for producing messages.
+- Consumer: The consumer is responsible for getting the producer's messages.
+- Broker: The broker acts as a bridge between the consumer and the producer because messages travel through the broker.
```

Comment:
```suggestion
- Producer: responsible for producing messages.
- Consumer: responsible for getting the producer's messages.
- Broker: acts as a bridge between the consumer and the producer because messages travel through the broker.
```